rllib/evaluate.py

```diff
@@ -334,34 +234,29 @@ def run(args, parser):
     config["evaluation_interval"] = 1
 
     # Rendering and video recording settings.
-    if args.no_render:
-        deprecation_warning(old="--no-render", new="--render", error=False)
-        args.render = False
-    config["render_env"] = args.render
+    config["render_env"] = render
```

Comment:
We need to redo the rendering options anyways. I think they broke after gym discontinued support for their rendering windows utils.

Reply:
I leave it as is for now. I think it makes sense to expose this argument, no matter how the internals currently look like.